### PR TITLE
only use stdout with "update" subcommand while not debugging

### PR DESCRIPTION
**Pull request by jakecoffman**
This PR restricts the use of stdout added in #78 to the `update` subcommand which was the original intent, but missed the mark.

Also using stdout while debugging messes up the terminal, so I've suppressed it in that case as well.

### CODE REVIEW

File: cmd/dependabot/internal/cmd/test.go [hunk 1]
```diff
@@ -49,6 +49,7 @@ var testCmd = &cobra.Command{
 			PullImages:    pullImages,
 			Timeout:       timeout,
 			UpdaterImage:  updaterImage,
+			UseStdout:     false,
```
Comment (landongrindheim) [inline cmd/dependabot/internal/cmd/test.go:52]: I'm not too familiar with the ergonomics of Golang, so please forgive my ignorance 😅 

Could we pass around the output file rather than a boolean?

**Review by landongrindheim — APPROVED** [review]
Suggested a change that would make sense _to me_ in Ruby. I'd love your feedback on that regardless of acceptance. I trust your judgment 🚀

File: cmd/dependabot/internal/cmd/test.go (hunk 1 above)
Comment (jakecoffman) [reply cmd/dependabot/internal/cmd/test.go:52]: That's fair, I knew this code needed a refactor from this but the one I made got a little too large so I went with the bool. Let me try again with an `io.Writer` and see how that goes. 

I think the RunParams is starting to turn into a code smell, eventually I'll get around to fixing that up!
